vipps-report-api-faq.md

```diff
@@ -33,6 +34,47 @@ We are still quite early in the planning and design phase and will use the
 draft documentation here to discuss with potential users of the API
 and make sure we are developing the right solution.
 
+## Which API keys give access to the API?
+
+In short: These API keys give access to the Report API:
```

Comment:
These -> The following

Reply:
I disagree, but I'll giver you this one.